modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase

```diff
@@ -973,6 +973,8 @@ definition {
 
 		var slaName = "SLAName";
 
+		WorkflowMetrics.executeIndexActions(actionDescription = "Workflow Indexes");
+
```

Comment:
For next time, let's commit changes in chunks rather than putting them all in one commit. For example, there could be a commit that says `Reindex to remove other Single Approvers` and `Temporarily add pause to stabilize test` @kristinonias